src/locale/pages/organizeJourneyInstances/en.yml

```diff
@@ -0,0 +1,10 @@
+columns:
+  id: Title
+  people: People
+  created_at: Created
```

Comment:
I understand what's causing this, and can see why it's annoying, but I think it's better to add these columns statically in the code rather than looping over the object, and in that case using custom names that adhere to the linting rules should not be a problem.